cit3.0-web/src/data/userStoryPaths.json

```diff
@@ -410,7 +410,7 @@
         "group": "what",
         "value": "3-COMPARING",
         "label": "comparing an area to another one",
-        "longText": "<div class='select-break'></div><p><b>Our tool lets you compare different types of areas to each other or to the B.C. average. You can select the types of area (such as Communities, Reserves, Regional Districts, Economic Regions, and much more) and the specific areas you want to compare.</b></p><div class='select-break'></div>",
+        "longText": "<div class='select-break'></div><p><b>Our tool lets you compare different types of areas to each other or to the B.C. average. You can select the types of area such as Census Subdivisions (Municipalities, Electoral Areas, First Nations Reserves), Tourism Regions, Economic Regions  and the specific area you want to compare.</b></p><div class='select-break'></div>",
```

Comment:
I think there is an extra space after Economic Regions... same with line 669

Reply:
good eyes thanks